projects/woff2/Dockerfile

```diff
@@ -20,4 +20,5 @@ RUN apt-get install -y make autoconf automake libtool
 
 RUN git clone --depth 1 --recursive https://github.com/google/woff2
 WORKDIR woff2
-COPY build.sh convert_woff2ttf_fuzzer.* $SRC/
+COPY build.sh *.options seed_corpus.zip $SRC/
```

Comment:
Ok, now I see! Wouldn't it be better to store testcases unpacked, as we do in Chromium?